components/ArticleCategories.js

```diff
@@ -40,8 +137,10 @@ function Category({ category }) {
     <>
       <Chip
         className={classes.category}
+        disabled={deletingCategory}
         label={category.title}
         onClick={handleOpen}
+        onDelete={canDelete ? () => deleteCategory() : undefined}
```

Comment:
why not just write `canDelete ? deleteCategory : undefined`

Reply:
I don't want DOM events being sent to mutation function, thus wrapping with arrow function

Reply:
Actually I did not test if `Chip`'s `onDelete` passes anything :P